### PR TITLE
Parse census data

### LINKED ISSUE #2: Parse given census data into database
Write a Python script to parse, clean, and store the census collection data from 2010 into the database. 

Cleaning steps we need to take include: 
- percentages that are not in the range [0, 1]

- data points that are not tracts (such as counties, etc)

We want to be able to store each data point as an object under a Mongo collection so that we can access it later. Create a Model for the Collection, and then do `.save()` to save btw.

Each data pt should have: Date (as mongo datetime obj), tractId, 2000 response rate, 2010 response rate

[Sample data](https://www2.census.gov/programs-surveys/decennial/2010/data/participation-rates-states/participationrates2010.txt?#)

### PR DESCRIPTION
**Pull request by coconut750750**
Resolves #2 
Added a Python script that takes a link and parses the census data into a Python object.

An example database object:
```
{
    "tract_id" : "17031311100", 
    "county" : "Cook County, IL",
    "rates" : { 
        "2000": {
            "03252000" : 0, 
        },
        "2010" : { 
            "03252010" : 0, 
            "03262010" : 0,
            "03292010" : 0,
            "03302010" : 0,
            "03312010" : 0,
            "04022010" : 0,
            "04052010" : 0,
            "04062010" : 0,
            "04072010" : 0,
            "04082010" : 0,
            "04092010" : 0,
            "04122010" : 0,
            "04132010" : 0.17,
            "04142010" : 0.17,
            "04152010" : 0.33,
            "04162010" : 0.33,
            "04192010" : 0.33,
            "04202010" : 0.5,
            "04212010" : 0.5,
            "04222010" : 0.5,
            "04232010" : 0.5,
            "04262010" : 0.5,
            "04272010" : 0.5
        }
    }
}
```

### CODE REVIEW

**Comment by chyku** [comment]
[Data over different dates](https://www2.census.gov/programs-surveys/decennial/2010/program-management/4-release/mail-participation-rate/)

**Comment by coconut750750** [comment]
Added a new endpoint, `/census_response`, to the backend to accept a batch of Census Response Entries to populate the database.

Updated the script to use that endpoint to save Census Responses.

File: backend/api/models/CensusResponse.py
Comment (chyku) [inline backend/api/models/CensusResponse.py]: By responseId, do you mean the tract Id? could we make this into a number-based field?

File: backend/api/models/CensusResponse.py
Comment (chyku) [inline backend/api/models/CensusResponse.py]: These both should just be Tract and tract, as those are the only ones we're concerned with

File: backend/api/views/main.py
Comment (chyku) [inline backend/api/views/main.py:56]: Could we put the webpage parsing into this endpoint? I think it would be pretty hard to make the webpage data into a json format

**Review by chyku — CHANGES_REQUESTED** [review]
Great work! I'm just clearing up some things I didn't clarify earlier, and it'd be great if you could put webpage parsing directly in the endpoint (which is p sus ik but idk if it'd be great to be parsing the page into a json file that you have to copy & paste manually)

Also I'm going to try to clean up the backend stuff so that there aren't a thousand changes

File: backend/api/models/CensusResponse.py
Comment (chyku) [inline backend/api/models/CensusResponse.py]: pls update comment

File: backend/api/views/web_scrap.py
Comment (chyku) [inline backend/api/views/web_scrap.py]: no newline- can u go to backend into venv and run `black .`

File: backend/api/views/populate_db.py
Comment (chyku) [inline backend/api/views/populate_db.py]: clean up unused comments?

**Review by caiyingshen — APPROVED** [review]
👻

looks good!! yay first PR!! 
(wait for kelley's approval before merging please)

**Review by chyku — CHANGES_REQUESTED** [review]
just tiny things

**Review by chyku — APPROVED** [review]

**Pull request merged**
